content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -40,6 +40,9 @@ spec:
   failedJobsHistoryLimit: 5                 # Optional. Default: 100. How many failed jobs should be kept.
   envSourceContainerName: {container-name}  # Optional. Default: .spec.JobTargetRef.template.spec.containers[0]
   maxReplicaCount: 100                      # Optional. Default: 100
+  scalingStrategy: "custom"                 # Optional. Default: default Which ScalingStrategy to use. 
+  customScalingQueueLengthDeduction: 1      # Optional. A parameter to optimize custom ScaleLogic.
```

Comment:
`# Optional. A parameter to optimize custom ScaleLogic.` -> `# Optional. A parameter to optimize custom Scale Strategy.`

Reply:
Done